components/automate-chef-io/data/docs/api-static/02-meta-description.yaml

```
@@ -6,7 +6,7 @@ info:
     To create admin token can and set it as an environment variable use the following command:
 
     ```
-    export TOKEN=`chef-automate admin-token`
+    export TOKEN='chef-automate admin-token'
```

This changes the meaning of this line.

In the shell programming language. The backtick operator executes an external command and returns it output.

That is, this line was previously saying:

"Run the command `chef-automate admin-token` and assign the output to the exported variable TOKEN"

With this change it would be saying:

"Assign the string 'chef-automate admin-token' to the exported variable Token"

If we would like to avoid confusion between the backtick operator and our documentation markup, then I recommend we use something like:

```suggestion
    export TOKEN=$(chef-automate admin-token)
```

---

Works OK on earlier A1 version; closing PR.